### DISCUSSION TITLE
pandoc's HTML has bad void elements

### DISCUSSION THREAD

**Discussion by reagle** [jgm/pandoc#9345] [Q&A] [answered]
I haven't kept up with HTML and I recently ran pandoc-generated HTML through a validator and it complains about terminating slashes in self-closing elements. Apparently, these elements, such as `<br>`, are now called "void" elements and "Trailing slash on void elements has no effect and interacts badly with unquoted attribute values."

Should the HTML template being adapted to this?

**Comment by jgm** [comment, marked as answer]
Yes, we want them there because we aim to produce "polyglot HTML" that is also valid XML.
They are also required for EPUB contents, which we generate using the HTML writer.
